webhook/user_info.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package webhook
+
+import (
+	"context"
+	"github.com/knative/pkg/apis"
+	"k8s.io/apimachinery/pkg/api/equality"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// CreatorAnnotationSuffix is the suffix of the annotation key to describe
+	// the user that created the resource.
+	CreatorAnnotationSuffix = "/creator"
+
+	// LastModifierAnnotationSuffix is the suffix of the annotation key to describe
+	// the user who last modified the resource.
+	LastModifierAnnotationSuffix = "/lastModifier"
```

Comment:
This annotation is changed from `xyz.knative.dev/updater` to `xyz.knative.dev/lastModifier`.

It was that way in eventing module and made more sense to me, thus I used it that way.

Happy to revert to `updater` if there are any objections.

Reply:
I believe this is used by client tooling, so it's a breaking change

Reply:
Yes, please don't change the label. It might actually be part of the spec.

Reply:
Ok, going to change it back to `/updater`

Reply:
Reverted to `/updater` with commits later.